resources/circuits_electronics.md

```diff
@@ -11,22 +11,286 @@
 
 ---
 
+## Introduction
+
+Recommended resources:
+
+Online public
+textbooks: [All About Circuits](https://www.allaboutcircuits.com/textbook/)
+
+(Paid)
+Textbook: [Sedra Smith Microelectronic Circuits](https://learninglink.oup.com/search/Sedra%20Smith%20Microelectronic%20Circuits?discipline=Engineering)
+
+---
+
 ## Intro to Circuits and Electronics
 
+---
+
 ## Ohms Law
 
+---
+
+| Quanity    | Symbol | Unit of Measurement |
+|------------|:------:|:-------------------:|
+| Current    |   I    |     Ampere (A)      |
+| Voltage    |   V    |      Volt (V)       |
+| Resistance |   R    |       Ohm (Ω)       |
+
+- The unit of resistance is ohms (1Ω = V/A)
+- Ohm's law says that the current through a conductor is given by the following;
+  $I = \frac{V}{R}$
+- Where V is the voltage across a conductor.
+
 ## KVL KCL
 
-## Voltage divider
+---
+
+### Kirchhoff’s Voltage Law (KVL)
+
+##### In a Series Circuit:
+
+- The algebraic sum of all voltages in loop must equal zero.
+- $V_{total} = V_{1} + V_{2} + V_{3} + ...V_{n}$
+
+##### In a Parallel  Circuit:
+
+- The voltage across every resistor is the same as the supply voltage.
+- $V_{total} = V_{1} = V_{2} = V_{3} = ...V_{n}$
+
+### Kirchhoff's Current law: (KCL)
```

Comment:
Stay consistent, previously you wrote a similar title as "Kirchhoff’s Voltage Law (KVL)".

Typo, in "Title Case" 'L' in "law" should be capitalized.